src/Pages/Tickets.jsx

```diff
@@ -1,22 +1,21 @@
 function Tickets (props) {
-const tickets = props.tickets
+const competitions = props.competitions
```

Comment:
This is fine, but you can make this even cleaner by destructing directly in the parameters. We know that `props` is an object, so we can do this:
`function Tickets({ competitions }) {`

That saves you a line of code so you don't have to do `const competitions = props.competitions`